client/src/controllers/DropdownController.ts

```diff
@@ -172,7 +178,8 @@ export class DropdownController extends Controller<HTMLElement> {
       trigger: 'click',
       interactive: true,
       ...(this.hasOffsetValue && { offset: this.offsetValue }),
-      getReferenceClientRect: () => this.getReference().getBoundingClientRect(),
+      hideOnClick: !this.useCustomHideOnClickAway,
+      getReferenceClientRect: () => this.reference.getBoundingClientRect(),
       theme: this.themeValue,
       plugins: this.plugins,
```

Comment:
Could have been clearer by letting themes define different plugins in `themeOptions`?